com.chartboost.mediation.demo/Assets/Samples/Chartboost Mediation/4.0.0/Chartboost Mediation/Editor/ChartboostMediationDependencies.xml

```diff
@@ -3,30 +3,29 @@
     <androidPackages>
         <!-- Helium SDK Maven Repo -->
         <repositories>
-            <repository>https://cboost.jfrog.io/artifactory/helium</repository>
+            <repository>https://cboost.jfrog.io/artifactory/chartboost-mediation/</repository>
         </repositories>
 
         <!-- Helium Android SDK -->
-        <androidPackage spec="com.chartboost:helium:4.0+" />
+        <androidPackage spec="com.chartboost:chartboost-mediation-sdk:4.0+" />
+        <androidPackage spec="androidx.legacy:legacy-support-v4:1.0.0" />
         <androidPackage spec="org.greenrobot:eventbus:3.3.1" />
+        <androidPackage spec="org.jetbrains.kotlin:kotlin-reflect:1.7.20" />
+        <androidPackage spec="org.jetbrains.kotlinx:kotlinx-coroutines-android:1.6.4" />
+        <androidPackage spec="org.jetbrains.kotlinx:kotlinx-serialization-json:1.4.1" />
+        <androidPackage spec="androidx.appcompat:appcompat:1.5.1" />
```

Comment:
I don't think we need this one. 🤔 As well as the Kotlin-reflect one. I may need to double-check. I'll make a note and check later.